python/src/dataset.rs

```diff
@@ -375,6 +377,42 @@ impl Dataset {
             Ok(None)
         }
     }
+
+    fn create_version_from_fragments(
+        self_: PyRef<Self>,
+        schema: &PyAny,
+        fragments: Vec<&PyAny>,
```

Comment:
who should be responsible for making sure all of the fragments have the same schema (and batch size?)

Reply:
feel that when we have some framework integration, we should do there?